apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector/shared.scss

```diff
@@ -0,0 +1,21 @@
+$template-selector-border-color: #e2e4e7;
+$template-selector-border-color-selected: #555d66;
+$template-selector-border-color-active: #00a0d2;
+$template-selector-border-color-hover: #c9c9ca;
+$template-selector-empty-background: #fff;
+$template-selector-modal-offset-bottom: 25px;
+$template-selector-modal-offset-right: 32px;
+$template-selector-blank-template-mobile-height: 70px;
+
+@mixin screen-reader-text() {
```

Comment:
Isn't this mixin either in Calypso or Gutenberg base styles already? Could we import it from there?

Reply:
I couldn't find it in Calypso upon a quick search. It appears to be in Gutenberg, https://github.com/WordPress/gutenberg/blob/bfd1117846c7802d62dbba046c27968d961781e7/packages/block-editor/src/components/responsive-block-control/style.scss#L1-L12, so we might be able to use that -- unless there's more implicit dependencies (like this: https://github.com/WordPress/gutenberg/pull/19159).

Overall, this PR is just moving around pre-existing styling; I'd like to limit changes that go beyond that in order to unblock it soon :grimacing:

Reply:
(We might actually want to use that entire component rather than just its styling.)

Reply:
Just moving things around in this PR makes sense. 👍